docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+## Coordination and support procedures
+
+### Coordination Approach
+
+FIWARE Lab activities are managed by weekly scrum meetings and monthly
```

Comment:
FIWARE Lab activities are managed by bi-weekly meetings on Tuesday 15:00 CET, 
all FIWARE Lab nodes administrators are invited to participate on it and share their 
status and discuss different topics relating to the activities carried out within the 
FIWARE Lab.